_pages/privacy.md

```diff
@@ -39,9 +39,10 @@ toc_sticky: true
 
 - We use a Google form to submit to the topic bazaar, but this is also stated
   separately [here]({% include fix-link.html link=submit_topic %}).
-- Participants register using a Google form for events.
+- Participants register via the [indico](https://indico.scc.kit.edu) platform
+  ([privacy statement](https://indico.scc.kit.edu/event/863/page/550-privacy-policy))
```

Comment:
```suggestion
- Participants register via the [indico](https://indico.scc.kit.edu/category/95/) platform
  using a dedicated registration link.
```
see https://github.com/SORSE/sorse.github.io/pull/229#issuecomment-659245451

Then let's remove the link to the privacy policy here